test_data/osm/catalog.yml

```diff
@@ -0,0 +1,6 @@
+sources:
+  osm_one_billion:
+    description: Test data points to same fake osm-3billion file
```

Comment:
Same as what?  Try to reword in a way that makes sense when reading just this one file.